src/routes/Home/components/Home.tsx

```diff
@@ -35,8 +35,11 @@ export default class Home extends React.PureComponent<IHomeProps,{
     this.props.updateType();
   }
 
-public onClick=()=> {
-  this.setState(state => ({ focusSelect: !state.focusSelect}))
```

Comment:
This line is required in order to focus the input box, if this line breaks the scrolling you can try using a setTimeout with 100ms : )